api/api_ethereum.go

```diff
@@ -543,6 +556,16 @@ func newEthRPCTransactionFromBlockIndex(b *types.Block, index uint64) *EthRPCTra
 	return newEthRPCTransaction(txs[index], b.Hash(), b.NumberU64(), index)
 }
 
+// newEthRPCTransactionFromBlockHash returns a transaction that will serialize to the RPC representation.
+func newEthRPCTransactionFromBlockHash(b *types.Block, hash common.Hash) *EthRPCTransaction {
```

Comment:
How about changing this name to `newEthRPCTransactionFromBlock`?
I misunderstood hash parameter is block hash first time.

Reply:
@jimni1222 
I think it would be great to use same function name with Ethereum source code.
Ethereum uses [newEthRPCTransactionFromBlockHash](https://github.com/ethereum/go-ethereum/blob/aaca58a7a1d9acbd24bbc74c49933efa2f1af183/internal/ethapi/api.go#L1373-L1381).

As you said it seems that hash may be misunderstood as meaning blockhash but, the function source codes are short and easy to understand so we don't have to worry much about this part.

Reply:
Um okay. If this is came from Ethereum code, then use same name might be better.